DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialVideoViewModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using System;
+    using DigitalLearningSolutions.Data.Models.TutorialContent;
+    using Microsoft.Extensions.Configuration;
+
+    public class TutorialVideoViewModel
+    {
+        public int CustomisationId { get; }
+        public int SectionId { get; }
+        public int TutorialId { get; }
+        public string TutorialName { get; }
+        public string CourseTitle { get; }
+        public string VideoPath { get; }
+
+        public TutorialVideoViewModel(
+            IConfiguration config,
+            TutorialVideo tutorialVideo,
+            int customisationId,
+            int sectionId,
+            int tutorialId
+        )
+        {
+            CustomisationId = customisationId;
+            SectionId = sectionId;
+            TutorialId = tutorialId;
+            TutorialName = tutorialVideo.TutorialName;
+            CourseTitle = tutorialVideo.CourseTitle;
+            VideoPath = GetVideoPath(config, tutorialVideo.VideoPath);
+        }
+
+        private static string GetVideoPath(IConfiguration config, string videoPath)
+        {
+            if (Uri.IsWellFormedUriString(videoPath, UriKind.Absolute))
+            {
+                return videoPath;
+            }
+            if (Uri.IsWellFormedUriString("https://" + videoPath, UriKind.Absolute))

```

Comment:
Minor but could extract https + url to variable to avoid creating twice 🙂 And I'd generally use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) instead of `+`. But that's very minor in these small cases, feel free to stick to `+` if you prefer 🙂

Reply:
Yeah I agree, done in 091daa2 🙂